### PR TITLE
Fix description for istioctl verify-install

### LINKED ISSUE #24068: 'istioctl verify-install' no longer does precheck
**Bug description**
Now I am trying to use Istio 1.6.0. But, When I run the command: istioctl verify-install . Then, It occurs some errors like this: Error: **could not load IstioOperator from cluster: the server could not find the requested resource.  Use --filename**。
**But Istio 1.5 is ok without error like above。**

My K8S information：
Version: 1.16

**Expected behavior**

**Steps to reproduce the bug**

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**

**How was Istio installed?**

**Environment where bug was observed (cloud vendor, OS, etc)**

### PR DESCRIPTION
**Pull request by jwendell**
Closes #24068.

### CODE REVIEW

File: istioctl/pkg/install/verify.go [hunk 1]
```diff
@@ -250,18 +250,21 @@ func NewVerifyCommand() *cobra.Command {
...
-		and report whether the cluster is ready for Istio installation.
+		If you do not specify an installation it will check for an IstioOperator resource
+		and will verify if pods and services defined in it are present.
+
+		Note: For verifying whether your cluster is ready for Istio installation, see
+		istioctl experimental precheck.
 `,
 		Example: `
-		# Verify that Istio can be freshly installed
+		# Verify that Istio is installed correctly via Istio Operator
```
Comment (linsun) [inline istioctl/pkg/install/verify.go:267]: Is this still correct without passing any param?
I'm getting this with latest master code base.
```
$ go run "${GOPATH}"/src/istio.io/istio/istioctl/cmd/istioctl/main.go verify-install                                                     

Error: could not load IstioOperator from cluster: control plane revision "" not found.  Use --filename
exit status 1
```

**Comment by linsun** [comment]
cc @irisdingbj

File: istioctl/pkg/install/verify.go (hunk 1 above)
Comment (jwendell) [reply istioctl/pkg/install/verify.go:267]: Yes, I'm getting the same. That's what motivated this PR. The description is outdated.
See https://github.com/istio/istio/issues/24068#issuecomment-634015870 and https://github.com/istio/istio/pull/23087#issue-406266978

**Comment by jwendell** [comment]
cc @esnible

**Comment by jwendell** [comment]
It changed in 1.6 :/

**Comment by irisdingbj** [comment]
Yep, behaviour changed for this command in 1.6 so I will agree for this change. But I want @esnible to confirm this as he made this changes.

File: istioctl/pkg/install/verify.go (hunk 1 above)
Comment (linsun) [reply istioctl/pkg/install/verify.go:267]: Ok, I see.  Can we clarify the msg?  Via Istio operator, does it mean server side operator or istioctl?  Do we need to specify istio operator here?

**Pull request merged**